### PR TITLE
[UWP] Fix column growing due to background image

### PR DESCRIPTION
**Pull request by almedina-ms**
Fixes https://github.com/microsoft/AdaptiveCards/issues/2697

The issue was fixed using TileControl to render the cover image and using the ITileBrush interface to position the image where it must be and to stretch it to fill

**There may be another issue where the background images make the card grow, this also fixes that**

### How verified 

The test app was run and the results were updated

### CODE REVIEW

File: samples/Tests/BackgroundImage.VerticalAndHorizontalAlignment.json [hunk 1]
```diff
@@ -114,7 +114,7 @@
 				},
 				{
 					"type": "Column",
-					"width": "auto",
```
Comment (paulcam206) [inline samples/Tests/BackgroundImage.VerticalAndHorizontalAlignment.json]: >th": "auto", [](start = 9, length = 12)

do we have other tests that cover the `"width": "auto"` case?

Comment (almedina-ms) [reply samples/Tests/BackgroundImage.VerticalAndHorizontalAlignment.json]: yes but the result looks better with stretch width as auto overflows the card in UWP and WPF

---
In reply to: [286154178](https://github.com/microsoft/AdaptiveCards/pull/2955#discussion_r286154178) [](ancestors = 286154178)

File: source/uwp/Renderer/lib/TileControl.cpp [hunk 2]
```diff
@@ -293,29 +302,47 @@ namespace AdaptiveNamespace
                 ComPtr<IUIElement> rectangleAsUIElement;
                 THROW_IF_FAILED(rectangleAsShape.As(&rectangleAsUIElement));
 
+                double originPositionX = (fillMode == BackgroundImageFillMode::Cover) ?  0.0 : (x * m_imageSize.Width) + offsetHorizontalAlignment;
+                double originPositionY = (fillMode == BackgroundImageFillMode::Cover) ? 0.0 : (y * m_imageSize.Height) + offsetVerticalAlignment;
```
Comment (paulcam206) [inline source/uwp/Renderer/lib/TileControl.cpp]: I think I'd rather see this implemented as an if/else (same below)

File: samples/Tests/BackgroundImage.VerticalAndHorizontalAlignment.json (hunk 1 above)
Comment (paulcam206) [reply samples/Tests/BackgroundImage.VerticalAndHorizontalAlignment.json]: that's fine -- I just wanted to make sure we were getting visual verification for the `auto` case :)

---
In reply to: [286155401](https://github.com/microsoft/AdaptiveCards/pull/2955#discussion_r286155401) [](ancestors = 286155401,286154178)

**Pull request merged**